.github/workflows/a-team.yml

```diff
@@ -8,6 +8,52 @@ on:
       - main
 
 jobs:
+  aws_setup:
```

Comment:
Lets name this job in a generic way. For example: infra_provisioning.

Reply:
addressed